src/App.vue

```diff
@@ -7,16 +7,12 @@
 
 <script>
 import GenreList from "@/components/GenreList.vue";
-import store from "@/store";
 
 export default {
   name: "App",
   components: {
     GenreList
   },
-  mounted() {
```

Comment:
i moved this call into the mounted of the List comp, because right now its not effecting App. and because i was doing the vuex map import in List.

but if the app grows you might wanna make this call from here.